File: Airsoft/src/app/models/orders/orderDetailsModel.ts
import { CourierOrderDetailsModel } from "../couriers/courierOrdetDetailsModel";
import { OrderGunViewModel } from "./orderGunViewModel";

export interface OrderDetailsModel{
    id: string,
    totalPrice: number,
    paymentType: string,
    orderStatus: string,
    guns: OrderGunViewModel[],
    courier: CourierOrderDetailsModel
}

File: serverAPI/AirsoftServer/ViewModels/Order/OrderDetailsModel.cs
namespace ViewModels.Order
{
    using System.Collections.Generic;

    using ViewModels.Courier;

    public class OrderDetailsModel
    {
        public string Id { get; init; }

        public decimal TotalPrice { get; init; }

        public CourierOrderViewModel Courier { get; init; }

        public ICollection<OrderGunsViewModel> Guns { get; init; }

        public string PaymentType { get; init; }

        public string OrderStatus { get; init; }
    }
}
